hackathon_site/dashboard/frontend/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -46,10 +60,20 @@ const Dashboard = () => {
         }
     }, [dispatch, hardwareInOrders]);
 
+    const openEditTeamModal = () => {
+        //dispatch(getCurrentTeam());
```

Comment:
remove comment

Reply:
also you don't need to brackets here
`const openEditTeamModal = () => dispatch(openTeamModalItem())`
is enough